src/components/MatchCards/MatchCards.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { Grid, Paper, Typography } from "@material-ui/core";
+import axios from "axios";
+import { Match } from "../../models";
+import { useStyles } from "../../useStyle";
+import { MatchCard } from "../MatchCard";
+
+export interface MatchCardsProps {
+  eventId: number;
+}
+
+export const MatchCards: FunctionComponent<MatchCardsProps> = ({ eventId }) => {
+  const classes = useStyles();
+  const [matches, setMatches] = useState<Match[] | []>([]);
+
+  useEffect(() => {
+    axios
+      .get<Match[]>("/api/matches", {
+        params: {
+          event: eventId,
+        },
+      })
+      .then((response) => {
```

Comment:
There is no error handling here

Reply:
Added error handling and loading icon in: 75569f4